WordPress/gutenberg
# Global styles revisions: integrate style book

## What?
Hopefully resolves https://github.com/WordPress/gutenberg/issues/55577

Part of:

- https://github.com/WordPress/gutenberg/issues/55776

Adds a button to toggle the Style Book in global styles revisions.

## Why?

So users can preview global styles revisions in the Style Book.

The Style Book displays more patterns and blocks and therefore affords users the possibility of previewing specific historical styles that may not appear in the currently-loaded template, e.g., changes to the Calendar block.

## How?
Adding a second "global-styles-revisions:style-book" view to the editor canvas container that is activated when the Revisions and Style Book buttons are active.

## Testing Instructions

1. In the Global Styles panel, open the revisions view. 
2. Test that the Style Book button toggles the Style Book. And that the styles update as you move through the revision history.
3. Check that the close button closes the Style Book view, but not revisions. A second click on the close button will exit revisions.
4. Now return to the Global Styles panel. Click on the Style Book icon first, then the Revisions icon. Both icons should be in a pressed state with Revisions open using the Style Book in the canvas container.


## Screenshots or screencast 


https://github.com/WordPress/gutenberg/assets/6458278/7299d389-e9ba-4cdd-84a6-e6f5699f9711
